client/dashboard/app/router/sites.tsx

```diff
@@ -186,10 +188,7 @@ export const siteLogsIndexRoute = createRoute( {
 export const siteLogsPhpRoute = createRoute( {
 	getParentRoute: () => siteLogsRoute,
 	path: 'php',
-	loader: async ( { params: { siteSlug } } ) => {
-		const site = await queryClient.ensureQueryData( siteBySlugQuery( siteSlug ) );
-		await queryClient.ensureQueryData( siteSettingsQuery( site.ID ) );
-	},
+	loader: ( { params: { siteSlug } } ) => preloadLogsData( siteSlug, LogType.PHP ),
```

Comment:
I'm not sure if this works. Currently, I am getting duplicated endpoints instead: the first one is from the preload and the second one is when I actually land in the Logs page:

<img width="599" height="102" alt="Image" src="https://github.com/user-attachments/assets/c57e5edc-0e26-4f15-a9dd-a60ee1c74027" />

It seems, the end timestamps are always different, so the cache is always missed.

Reply:
Good catch. I think it's best to remove this from this PR for now, there is more work to do with timezones and times generally (will be dealt with in another PR) and ensuring this works with it adds to the complexities